packages/react-composites/tests/browser/common/fixtureHelpers.ts

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  CONNECTION_STRING,
+  CHAT_TOPIC_NAME,
+  ChatUserType,
+  CallUserType,
+  MeetingUserType,
+  PAGE_VIEWPORT
+} from './defaults';
+import { ChatClient } from '@azure/communication-chat';
+import { CommunicationIdentityClient, CommunicationUserToken } from '@azure/communication-identity';
+import { AzureCommunicationTokenCredential } from '@azure/communication-common';
+import { Browser, Page } from '@playwright/test';
+import { v1 } from 'uuid';
+import { buildUrl } from './utils';
+
+/**
+ * Creates a page to be tested for each participant in a browser page.
+ * To be used in a playwright fixture's 'pages'.
+ */
+// eslint-disable-next-line no-empty-pattern, @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/explicit-function-return-type
+export const usePagePerParticipant = async ({ serverUrl, testBrowser, users }, use) => {
+  const pages = await Promise.all(users.map(async (user) => loadPage(testBrowser, serverUrl, user)));
+  await use(pages);
```

Comment:
Isn't it grand that React and Playwright give wildly different meanings to `use`...